OutOfSchool/OutOfSchool.WebApi/Controllers/V1/WorkshopController.cs

```diff
@@ -263,6 +263,7 @@ public async Task<IActionResult> GetByFilter([FromQuery] WorkshopFilter filter,
     /// <response code="403">If the user has no rights to use this method, or sets some properties that are forbidden.</response>
     /// <response code="500">If any server error occures.</response>
     [HasPermission(Permissions.WorkshopAddNew)]
+    [Authorize]
```

Comment:
can you double check that `HasPermission` does not work as authorize as it extends authorize? Maybe it is better to fix HasPermission instead of duplicating attributes